setup/jetstream/nats.go

```diff
@@ -53,7 +53,7 @@ func (s *NATSInstance) Prepare(process *process.ProcessContext, cfg *config.JetS
 		if err != nil {
 			panic(err)
 		}
-		s.ConfigureLogger()
+		s.SetLogger(NewLogAdapter(), false, false)
```

Comment:
Maybe worth to explain that `false, false` is `debugFlag` and `traceFlag`

Reply:
Changed this to use the values from the NATS options which mimics the exact behavior of ConfigureLogger which was used before.